app/src/app/navtitle/navtitle.component.html

```diff
@@ -3,33 +3,61 @@ <h1 i18n><a routerLink="/">Itero</a></h1>
   <h2 i18n>Online iterative vote <span class="long-title">application</span></h2>
   <nav>
     <a *ngIf="(demoIsActive$ | async) && !(session.state$ | async).logged"
-      [routerLink]="(demoPath$ | async)" routerLinkActive="active">
+      [routerLink]="(demoPath$ | async)" routerLinkActive="active" class="nav-link">
       <span class="navitem" i18n>Try it</span>
     </a>
-    <a *ngIf="(session.state$ | async).logged" routerLink="/r/list" routerLinkActive="active">
+    <a *ngIf="(session.state$ | async).logged" routerLink="/r/list" routerLinkActive="active" class="nav-link">
       <span class="navitem" i18n>Polls</span>
     </a>
-    <a *ngIf="!(session.state$ | async).logged" routerLink="/r/session/signup" routerLinkActive="active">
+    <a *ngIf="!(session.state$ | async).logged" routerLink="/r/session/signup" routerLinkActive="active" class="nav-link">
       <span class="navitem" i18n>Sign up</span>
     </a>
-    <a *ngIf="!(session.state$ | async).logged" routerLink="/r/session/login" routerLinkActive="active">
+    <a *ngIf="!(session.state$ | async).logged" routerLink="/r/session/login" routerLinkActive="active" class="nav-link">
       <span class="navitem" i18n>Log in</span>
     </a>
-    <a *ngIf="(session.state$ | async).logged"  routerLink="/r/create" routerLinkActive="active">
+    <a *ngIf="(session.state$ | async).logged" routerLink="/r/create" routerLinkActive="active" class="nav-link">
       <span class="navitem">Create</span>
     </a>
-    <a *ngIf="(session.state$ | async).logged" [matMenuTriggerFor]="profileMenu">
+    <a [matMenuTriggerFor]="profileMenu" id="naviconmenu">
```

Comment:
Names are easier to read and remember if the words are separated. With CSS, the usage is to separate words using `-`. Hence naviconmenu  would be more readable if it was nav-icon-menu. Same for nav-icon-person below. I haven't done that for all class and id names and I regret it. Notice too that the elements on which these id are applied are **not** icons, which can be misleading.